packages/web-components/src/components/locale-modal/region-item.ts

```diff
@@ -49,8 +55,15 @@ class C4DRegionItem extends C4DCard {
   protected _renderDisabledLink() {
     const { _classes: classes, _cardClasses: cardClasses } = this;
     return html`
-      <button id="link" class="${classes}" disabled type="button">
-        <div class="${cardClasses}">${this._renderInner()}</div>
+      <button
+        id="link"
+        class="${classes}"
+        disabled
+        type="button"
+        part="button disabled-button">
```

Comment:
Lets follow a convention we've been using elsewhere:
```suggestion
        part="button button--disabled">
```